cloud/ec2.go

```diff
@@ -318,13 +318,6 @@ func (m *ec2Manager) spawnOnDemandHost(ctx context.Context, h *host.Host, ec2Set
 		input.UserData = &userData
 	}
 
-	grip.Debug(message.Fields{
```

Comment:
Remove this one + the one below doesn't change traceability because the provisioning create host job already logs when it successfully creates a host, and the EC2 manager already `grip.Error`s in all the error cases.